tests/linalg_test.py

```diff
@@ -45,10 +45,8 @@ def float_types():
              for dtype in [onp.float32, onp.float64])
 
 def complex_types():
-  return {onp.complex64}
-  # TODO(phawkins): change to the following after another jaxlib release.
-  #return set(onp.dtype(xla_bridge.canonicalize_dtype(dtype))
-  #           for dtype in [onp.complex64, onp.complex128])
+  return set(onp.dtype(xla_bridge.canonicalize_dtype(dtype))
```

Comment:
Nit driven by my own curiosity about Python:

You could use a set comprehension here:

```python
return {onp.dtype(xla_bridge.canonicalize_dtype(dtype))
        for dtype in [onp.complex64, onp.complex128]}
```

I checked using `dis.dis` that it actually generates different bytecode:

```
In [1]: from dis import dis

In [2]: def f():
   ...:     return {x for x in [1, 2]}
   ...:

In [3]: dis(f)
  2           0 LOAD_CONST               1 (<code object <setcomp> at 0x7f001eb86730, file "<ipython-input-2-78b46c213eea>", line 2>)
              3 MAKE_FUNCTION            0
              6 LOAD_CONST               2 (1)
              9 LOAD_CONST               3 (2)
             12 BUILD_LIST               2
             15 GET_ITER
             16 CALL_FUNCTION            1
             19 RETURN_VALUE

In [4]: def g():
   ...:     return set(x for x in [1, 2])
   ...:

In [5]: dis(g)
  2           0 LOAD_GLOBAL              0 (set)
              3 LOAD_CONST               1 (<code object <genexpr> at 0x7f001eb6ff30, file "<ipython-input-4-3a50cfee158d>", line 2>)
              6 MAKE_FUNCTION            0
              9 LOAD_CONST               2 (1)
             12 LOAD_CONST               3 (2)
             15 BUILD_LIST               2
             18 GET_ITER
             19 CALL_FUNCTION            1
             22 CALL_FUNCTION            1
             25 RETURN_VALUE
```